spec/features/01_user_sign_in_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+feature "sees sign in options" do
+  scenario "user visits index page" do
+    visit '/'
+    expect(page).to have_content('Sign In')
+    expect(page).to have_content('Sign Out')
+  end
+
+  scenario "user succesfully signs in" do
+    user = FactoryGirl.create(:user)
```

Comment:
Consider how some of this logic could be moved into a `before` block, or what contexts these scenarios might belong to.